content/TrueNASUpgrades/_index.md

```diff
@@ -1,29 +1,97 @@
 ---
-title: "Release Schedules"
+title: "Software Releases"
 description: "Centralized schedules and upgrade charts for software releases."
 weight: 25
 related: false
+aliases:
+ - /scale/gettingstarted/upgrades/
 ---
 
```

Comment:
I think we need at least an h2 here, if not maybe a sentence or two letting readers know that these tables contain links to download different build versions. The sidebar toc looks a little odd with the first two items being indented h3s also, but adding a header at the top would help with that too